src/main/app/claims/claimModelConverter.ts

```diff
@@ -26,6 +27,7 @@ import { StatementOfTruth } from 'claims/models/statementOfTruth'
 import { StringUtils } from 'utils/stringUtils'
 import { InterestType } from 'claim/form/models/interest'
 import { ClaimantTimeline } from 'claim/form/models/claimantTimeline'
+import { Payment } from 'app/pay/Payment'
```

Comment:
Please do not import through `app`, using `pay` alias should be enough (add a mapping in `tsconfig.json` if it's not already defined).

Reply:
Applies to all imports in this PR.

Reply:
I decided to use `payment-hub-client/` for now. Later I plan to publish payment hub client NPM module  which would be like `@hmcts/payment-hub-client`.